lib/src/features/wine_tasting_create_view/components/characteristics/wine_characteristics_section.dart

```diff
@@ -48,98 +50,230 @@ class CharacteristicsSection extends StatefulWidget {
-      child: Column(
-        children: [
-          SectionTitle(sectionNumber: 2, title: 'Characteristics'),
-          SizedBox(height: 20),
-          Text(
-            'Tap or drag to mark the intensity of a characteristic on a scale of zero to six.',
-            style: Theme.of(context).textTheme.caption,
-            textAlign: TextAlign.center,
-          ),
-          SizedBox(height: 30),
-          Expanded(
-            child: ListView.separated(
-              // Disable vertical scrolling so that it doesn't interfere with dragging horizontally.
-              physics: const NeverScrollableScrollPhysics(),
-
-              itemBuilder: (context, index) {
-                String name = characteristics[index]['name'];
-                double value = characteristics[index]['value'];
-                var onChanged = characteristics[index]['on_changed'];
-
-                return Column(
-                  children: [
-                    Row(
-                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                      children: [
-                        Text(
-                          name.toUpperCase(),
-                          textAlign: TextAlign.start,
-                          style: Theme.of(context).textTheme.overline,
-                        ),
-                        Text('${value} / 6.0', style: Theme.of(context).textTheme.subtitle2),
-                      ],
+      padding: const EdgeInsets.symmetric(vertical: 10.0),
+      child: StreamBuilder(
+        stream: wineTastingBloc.sliders,
+        builder: (BuildContext context, AsyncSnapshot<List<CustomSlider>> snapshot) {
+          if (snapshot.hasData) {
+            var sliders = snapshot.data;
+
+            return Column(
+              children: [
+                SectionTitle(sectionNumber: 2, title: 'Characteristics'),
+                SizedBox(height: 20),
+                Padding(
+                  padding: const EdgeInsets.symmetric(horizontal: 30.0),
+                  child: Text(
+                    'Tap or drag to mark the intensity of a characteristic on a scale of zero to six.',
+                    style: Theme.of(context).textTheme.caption,
+                    textAlign: TextAlign.center,
+                  ),
+                ),
+                SizedBox(height: 30),
+                Expanded(
+                  child: Scrollbar(
+                    isAlwaysShown: true,
+                    child: ListView.separated(
+                      // Disable vertical scrolling so that it doesn't interfere with dragging horizontally.
+                      // physics: const NeverScrollableScrollPhysics(),
+
+                      itemBuilder: (context, index) {
+                        // var onChanged = characteristics[index]['on_changed'];
+
+                        var characteristicName = sliders[index].name;
+
+                        var initialValue;
+                        if (wineTastingBloc.characteristics != null &&
```

Comment:
If `wineTastingBloc.characteristics` is never null, remove the check and simply set the value, allow `?? = 0` to handle when the value is null